start_chromium_with_proxy.sh

```
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+if [ ! -d $HOME/.pki/nssdb ]; then
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)